A new term entry for the padEnd string method in javascript (#4748)
Codecademy/docs: content/javascript/concepts/strings/terms/padEnd/padEnd.md

> ---
> Title: '.padEnd()'
> Description: 'Pads a string with a given string several times until it reaches the given length.'
> Subjects:
>   - 'Computer Science'
>   - 'Web Development'
> Tags:
>   - 'Strings'
>   - 'Methods'
> CatalogContent: 
>   - 'introduction-to-javascript'
>   - 'paths/front-end-engineer-career-path'
> ---

Reviewer: ```suggestion
---
Title: '.padEnd()'
Description: 'Pads a string with a given string until the resulting string reaches the specified length.'
Subjects:
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Methods'
  - 'Strings'
CatalogContent: 
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
---
```

In JavaScript, the **`padEnd()`** method is used with javascript strings to pad the end of a string with another string, until it reaches a specified length. It was introduced with ECMAScript 2017.

## Syntax

```pseudo

> string.padEnd(length, padStr)

Reviewer: ```suggestion
string.padEnd(targetLength, padString)
```

> ```pseudo
> string.padEnd(length, padStr)
> ```

Reviewer 2: ```suggestion
```pseudo
string.padEnd(length, padStr)
```
```

Author: I don't understand what the difference is here? @SaviDahegaonkar

> The string is a given variable containing a sentence, the length parameter is the value the string needs to reach and padStr is the string character to add to the given string until it reaches the desired length. >**Note** if you wish to use this method with a number, you must convert to a string.

Reviewer: Avoid using this description, rather describe the parameters used in the above syntax in the following format:

- `string`: describe this parameter
- `targetLength`: describe this parameter
- `padString`: describe this parameter

> The string is a given variable containing a sentence, the length parameter is the value the string needs to reach and padStr is the string character to add to the given string until it reaches the desired length. Note if you wish to use this method with a nukber, you will need to convert to a string first.

Reviewer 2: ``` suggestion
The string is a given variable containing a sentence, the length parameter is the value the string needs to reach and padStr is the string character to add to the given string until it reaches the desired length. >**Note:** If you wish to use this method with a number, you must first convert to a string.
```
At the time of writing entries, the note should always be written using the blockquote(>) and also use an asterisk symbol like this(**). To understand it properly, you can get more information about this [at](https://github.com/Codecademy/docs/tree/main/documentation).

## Example

> In the following example, a string variable called `text` contains a sentence. The `padEnd()` method is used to pad the text out to the specified length.

Reviewer: ```suggestion
In the following example, a string variable called `text` contains the sentence 'hello world'. The `.padEnd()` method is then used to append additional characters (in this case, exclamation marks '!') to the end of the text until it reaches the specified length of 15 characters.
```

> In the following example, a string variable called `text` contains a sentence. The `padEnd()` method is used to pad the text out to the length provided.

Reviewer 2: ```suggestion
In the following example, a string variable called `text` contains a sentence. The `padEnd()` method is used to pad the text out to the specified length.
```

```js

> let text = 'hello world'
> text.padEnd('15', '!')

Reviewer 2: ```suggestion
let text = 'hello world'
console.log(text.padEnd(15, '!'));
```
In the above code to print the output, we should use the console.log statement in Javascript.
And for syntax, the `length` parameter should be a number, not a string, I have corrected it accordingly. Keep a note of this.

Author: I completely missed that, but notd thank you.

```

This results in the following output:

```shell

> 'hello world!!!!'

Reviewer 2: ```suggestion
'hello world!!!!'
```

Author: Again what is the difference I see no clear changes here @SaviDahegaonkar ? Is it spacing?

```

## Codebyte Example

> ```codebyte/js

Reviewer: ```suggestion
```codebyte/javascript
```

> ##Example runnable code block.

Reviewer: ```suggestion
```

> # Example runnable code block.
> let myStr = 'This is a testing string'
> console.log(myStr.padEnd(30, '!'));

Reviewer 2: ```suggestion
##Example runnable code block.
let myStr = 'This is a testing string'
console.log(myStr.padEnd(30, '!'));
```

> console.log(myStr.padEnd(30, '!'));

Reviewer: Instead of an exclamation mark (!), could you provide some other example here, since the above example also has the same structure?

Author: Good point I have changed this
